swiss_locator/swiss_locator_plugin.py

```diff
@@ -33,6 +33,11 @@
     SwissLocatorFilterLocation,
 )
 from swiss_locator.core.filters.swiss_locator_filter_wmts import SwissLocatorFilterWMTS
+try:
+    from swiss_locator.core.profiles.profile_generator import SwissProfileSource
+except ImportError:
+    # Should fail only for QGIS < 3.26, where profiles weren't available
```

Comment:
```suggestion
    # Should fail only for QGIS <= 3.36, where profiles weren't available
```

Reply:
In this case, it's 3.26, since that's the QGIS version where Profile functionality landed.
However, I'm checking against 3.37, because before 3.37 we cannot really register any `QgsAbstractProfileSource` subclass.